packages/react-app/src/contexts/BridgeContext.jsx

```diff
@@ -121,6 +124,25 @@ export const BridgeProvider = ({ children }) => {
     setLoading(false);
   }, [fromToken, ethersProvider, fromAmount, setToken]);
 
+  const setDefaultTokenList = useCallback(async (chainId, customTokens) => {
+    setLoading(true);
+    try {
+      const gotTokenList = await getTokenList(chainId);
+      gotTokenList.concat(customTokens.filter(token => token.chainId === chainId ))
```

Comment:
@scottrepreneur since we are concatenating this could lead to duplicates?

Reply:
Just noticed this. The concatenated list is not being set to the variable again, so it is just lost. 
And the duplicates issue needs to be fixed. 
I added the same token twice using the modal, and now it appears twice in the list

Reply:
Yeah good point. We could prevent duplicates from being saved or check for dupes at load.